### PR TITLE
tx detail page develop

### PR DESCRIPTION
**Pull request by wenwu-bianjie**
https://github.com/irisnet/explorer/issues/882

### CODE REVIEW

File: backend/model/tx.go [hunk 1]
```diff
@@ -208,6 +226,45 @@ func (s Signer) String() string {
 	return fmt.Sprintf("AddrHex: %v   AddrBech32: %v \n", s.AddrHex, s.AddrBech32)
 }
 
+type MsgItem struct {
+	Type    string  `bson:"type"`
+	MsgData MsgData `bson:"msg"`
+}
+
+type MsgData struct {
```
Comment (kaifei-bianjie) [inline backend/model/tx.go]: define different msgData struct by msg type

File: backend/model/tx.go [hunk 2]
```diff
@@ -208,6 +226,45 @@ func (s Signer) String() string {
...
+	TokenId        string `bson:"token_id"`
+	To             string `bson:"to"`
+	Family         string `bson:"family"`
+	Source         string `bson:"source"`
+	Gateway        string `bson:"gateway"`
+	Symbol         string `bson:"symbol"`
+	SymbolAtSource string `bson:"symbol_at_source"`
```
Comment (kaifei-bianjie) [inline backend/model/tx.go]: `symbol_at_source` and `symbol_min_alias` is no longer used

File: backend/model/tx.go (hunk 1 above)
Comment (wenwu-bianjie) [reply backend/model/tx.go]: have rewirete

File: backend/model/tx.go (hunk 2 above)
Comment (wenwu-bianjie) [reply backend/model/tx.go]: deleted

File: backend/orm/document/tx.go [hunk 3]
```diff
@@ -139,35 +116,42 @@ type Msg interface {
 }
 
 type MsgItem struct {
-	Type    string  `json:"type" bson:"type"`
-	MsgData MsgData `json:"msg" bson:"msg"`
+	Type    string  `bson:"type"`
+	MsgData MsgData `bson:"msg"`
```
Comment (kaifei-bianjie) [inline backend/orm/document/tx.go]: define `MsgData` to []interface{} and define tx msgs in document package

**Pull request merged**